tests/system/.classpath

```diff
@@ -59,5 +52,105 @@
+			<attribute name="gradle_used_by_scope" value="testapp4"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="src" output="bin/testapp5" path="src/testapp5/java">
+		<attributes>
+			<attribute name="gradle_scope" value="testapp5"/>
+			<attribute name="gradle_used_by_scope" value="testapp5"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="src" output="bin/testapp6" path="src/testapp6/java">
+		<attributes>
+			<attribute name="gradle_scope" value="testapp6"/>
+			<attribute name="gradle_used_by_scope" value="testapp6"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="src" output="bin/testapp6" path="src/testapp6/resources">
+		<attributes>
+			<attribute name="gradle_scope" value="testapp6"/>
+			<attribute name="gradle_used_by_scope" value="testapp6"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="src" output="bin/testapp7" path="src/testapp7/java">
+		<attributes>
+			<attribute name="gradle_scope" value="testapp7"/>
+			<attribute name="gradle_used_by_scope" value="testapp7"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="src" output="bin/testapp7" path="src/testapp7/resources">
+		<attributes>
+			<attribute name="gradle_scope" value="testapp7"/>
+			<attribute name="gradle_used_by_scope" value="testapp7"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="src" output="bin/testscriptapp1" path="src/testscriptapp1/java">
+		<attributes>
+			<attribute name="gradle_scope" value="testscriptapp1"/>
+			<attribute name="gradle_used_by_scope" value="testscriptapp1"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="src" output="bin/testscriptapp1" path="src/testscriptapp1/resources">
+		<attributes>
+			<attribute name="gradle_scope" value="testscriptapp1"/>
+			<attribute name="gradle_used_by_scope" value="testscriptapp1"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="src" output="bin/testscriptapp2" path="src/testscriptapp2/java">
+		<attributes>
+			<attribute name="gradle_scope" value="testscriptapp2"/>
+			<attribute name="gradle_used_by_scope" value="testscriptapp2"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="src" output="bin/testscriptapp2" path="src/testscriptapp2/resources">
+		<attributes>
+			<attribute name="gradle_scope" value="testscriptapp2"/>
+			<attribute name="gradle_used_by_scope" value="testscriptapp2"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11/"/>
```

Comment:
I don't think this is right. No need to specify Java 11, just use the configured jdk as in all other projects.